sources/osg/Matrix.js

```diff
@@ -1184,6 +1441,29 @@ define( [
             return vectorsArray;
         },
 
+        // better precison
+        // no far clipping artifacts.
+        // no reason not to use.
+        // Tightening the Precision of Perspective Rendering
+        //http://www.geometry.caltech.edu/pubs/UD12.pdf
+        // drop-in, just remove the one below, and rename this one
+        makeInfiniteFrustum: function ( left, right, bottom, top, znear, zfar, result ) {
```

Comment:
I would prefer to keep method start the same way, like makeFrustumInfinite or something like that, it's more convenient.